sirepo/feature_config.py

```diff
@@ -107,6 +107,10 @@ def b(msg, dev=False):
             msg,
         )
 
+    def _raydata_data_dir(path):
+        from sirepo import srdb
+        return srdb.root().join(path)
```

Comment:
aee9543ac

Can you define `data directory` and `global output directory`?

Reply:
The data directory is "input", that is, the XPCS files. Maybe they don't exist, and it's Bluesky that delivers all the input files directly to the notebook. My impress

The global output directory is where the notebook outputs its results. We are using those newly created directories as a mutex. These directories may be large, I believe, so they might not fit in srdb.root. We cannot symlink the directory  into srdb.root, because we have to mount it inside a container so the actual directory must be specified in the config (or I guess read from the symlinks, but this is annoying code to have write).

Reply:
The only input files currently are the mask files. Those are in the run_dir and unzipped into the output directory 377f2efca